keck_vnc_launcher.py

```diff
@@ -1067,17 +1179,7 @@ def validate_ssh_key(self):
 
         self.ssh_key_valid = False
         cmd = 'whoami'
-#         server = self.servers_to_try[0]
-
-        # Find a server in the servers_to_try list with is not an svncserverN
-        server = None
-        for s in self.servers_to_try:
```

Comment:
Why is this block being removed?  I think it is still necessary (though only in the case where the API is not functioning of course).

Reply:
I can’t even remember editing that function, so I think it was a rogue edit that didn’t get caught in my testing b/c it still works.  I’ll put it back to what it was.